python/paddle/fluid/initializer.py

```diff
@@ -30,1139 +30,24 @@
 from paddle import _C_ops, _legacy_C_ops
 import paddle
 
-__all__ = [
-    'Constant',
-    'Uniform',
-    'Normal',
-    'TruncatedNormal',
-    'Xavier',
-    'Bilinear',
-    'MSRA',
-    'ConstantInitializer',
-    'UniformInitializer',
-    'NormalInitializer',
-    'TruncatedNormalInitializer',
-    'XavierInitializer',
-    'BilinearInitializer',
-    'MSRAInitializer',
-    'NumpyArrayInitializer',
-    'set_global_initializer',
-]
+__all__ = ['set_global_initializer']
```

Comment:
why only `set_global_initializer` has not been moved or cleaned?

Reply:
There is a problem of circular dependency, imported by layer_helper_base. So it cannot be moved temporarily.